config/config.exs

```diff
@@ -53,3 +53,13 @@ config :guardian, Guardian.DB,
   token_types: ["access"],
   # default: 60 minute
   sweep_interval: 60
+
+# Configure :ex_aws
+config :ex_aws,
+  access_key_id: {:system, "AWS_ACCESS_KEY_ID"},
+  secret_access_key: {:system, "AWS_SECRET_ACCESS_KEY"},
```

Comment:
Add `:instance_role` so that in production, `ex_aws` will look for instance role credentials in our ec2 instances. Otherwise, this current configuration will fail in production.